src/client/src/rt-components/tear-off/TearOff.tsx

```diff
@@ -82,10 +100,9 @@ const TearOff: React.FC<TearOffProps> = ({ render, externalWindowProps, tornOff,
     return (
       <DragWrapper
         draggable={allowTearOff}
-        onDragEnd={(event: React.DragEvent<HTMLDivElement>) => {
-          popOut(event.screenX, event.screenY)
-        }}
-        onDragStart={onDragStart}
+        onDragEnd={(event: React.DragEvent<HTMLDivElement>) => popOut(event.screenX, event.screenY)}
+        onDragStart={createDragImage}
+        onMouseDownCapture={onMouseDownCapture}
```

Comment:
this is a new event where we will save coordinates of mouse relative to the tile. These coordinates will be used when we tearing things off